src/dsql/ExprNodes.cpp

```diff
@@ -3481,24 +3481,30 @@ dsc* BoolAsValueNode::execute(thread_db* tdbb, Request* request) const
 //--------------------
 
 
-static RegisterNode<CastNode> regCastNode({blr_cast});
+static RegisterNode<CastNode> regCastNode({blr_cast, blr_cast_format});
 
-CastNode::CastNode(MemoryPool& pool, ValueExprNode* aSource, dsql_fld* aDsqlField)
+CastNode::CastNode(MemoryPool& pool, ValueExprNode* aSource, dsql_fld* aDsqlField, const string& format)
 	: TypedNode<ValueExprNode, ExprNode::TYPE_CAST>(pool),
 	  dsqlAlias("CAST"),
 	  dsqlField(aDsqlField),
 	  source(aSource),
 	  itemInfo(NULL),
+	  format(pool, format),
```

Comment:
I don't personally agree, but we don't use parameters and fields with same name, so please do like the others and rename the parameter to `aFormat`.